patterns/testimonials-three-columns-six-testimonials.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Title: Three column layout with 6 testimonials
+ * Slug: twentytwentyfive/testimonials-three-columns-six-testimonials
+ * Keywords: testimonial
+ * Categories: testimonials
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+
+<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|60","bottom":"var:preset|spacing|60"},"blockGap":"var:preset|spacing|50"}},"layout":{"type":"constrained"}} -->
```

Comment:
This group should be set to full, so that if we add a background color it looks good.
![Captura de ecrã 2024-08-30, às 16 57 34](https://github.com/user-attachments/assets/23af640e-2247-4f84-aa87-a8b74dc41d5f)

Reply:
Thanks, updated in https://github.com/WordPress/twentytwentyfive/pull/118/commits/1467bbe13c4e8b970f30ca9aba2c23eee46e4009